react.js

```diff
@@ -31,15 +31,10 @@ const generateCacheKey = value => {
 }
 
 const customRender = (node, options = {}) => {
-  const locale = getLocale(path(['locale'], options))
-  const messages = locale
-    ? require(paths.resolveAppPath(`../messages/${locale}.json`))
-    : {}
-
-  const intlProps = {
-    locale: locale || 'en',
-    messages: messages,
-  }
+  const locale = getLocale(path(['locale'], options)) || 'en'
+  const messages = options.messages || require(paths.resolveAppPath(`../messages/${locale}.json`))
+
+  const intlProps = { locale, messages, textComponent: 'span' }
```

Comment:
shouldn't we use a fragment for the text component? so it's more aligned with what render-runtime does

Reply:
I think so. I've used span because our snapshots are using span.

Reply:
I've just changed to fragment. Keep in mind some snapshots will break though